stacks.yml

```diff
@@ -277,6 +277,9 @@ admin_frontend:
     EnvVarDmApiUrl: "{{ stacks.api.outputs.URL }}"
     EnvVarDmS3DocumentBucket: "{{ stacks.documents_s3.outputs.Name }}"
```

Comment:
It is at the moment but once this change goes in the admin app can be updated and then `EnvVarDmS3DocumentBucket` can be removed.